src/components/Letters/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import styled, { css } from "styled-components";
+import chroma from "chroma-js";
+
+import "./styles.css";
+
+export const Item = styled.div`
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');  
+display: flex;
+  justify-content: center;
+  font-family: 'Quicksand';
+  padding: .2rem;
+  ${({ color = chroma.random() }) =>
+    css `background-color: ${color};
+  color: ${chroma.contrast(color, "black") >= 4 ? "black" : "white"};
+  font-size: 2em;
+  font-weight: bold;
+  border: 2px dashed; 
+  border-color: ${color};
+  border-radius: 15px;
+  background-clip: content-box;
+  `}
+  `;
+
+export const Grid = styled.div`
```

Comment:
Okay, this looks nice and maintainable. Not like most grid implementations I've had to deal with. I'm not exactly sure how this will work with what @kndshein set up for the dynamic letters, but it seems like it shouldn't be too hard to merge the two together.